packages/esm-patient-medications-app/src/medications-summary/medications-summary.component.tsx

```diff
@@ -12,9 +12,11 @@ export interface MedicationsSummaryProps {
 
 export default function MedicationsSummary({ patientUuid }: MedicationsSummaryProps) {
   const { t } = useTranslation();
-  const [activePatientOrders] = usePatientOrders(patientUuid, 'ACTIVE');
+  const [activePatientOrders, fetchActivePatientOrders] = usePatientOrders(patientUuid, 'ACTIVE');
   const [pastPatientOrders] = usePatientOrders(patientUuid, 'any');
-
+  useEffect(() => {
+    fetchActivePatientOrders();
+  });
```

Comment:
This seems wrong to me - the effect would refetch the dependencies on every single render of the component. That is way too often. When exactly do you want to (re)fetch the orders? If it's only once during the initial render, you should add an empty array `[]` to the `useEffect`'s deps.